bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -317,6 +317,7 @@ export class HasProps extends Backbone.Model
           immediate = v._immediate_references()
           for obj in immediate
             HasProps._value_record_references(obj, result, true) # true=recurse
+    else if v?.buffer instanceof ArrayBuffer
```

Comment:
`?` is unnecessary here. There's already a check for `null` in the first clause. Does it happen to `v` to be `undefined` as well? If so, the first clause should be rewritten as `v?`.